tests/drivers/sdhc/boards/max32666fthr_max32666_cpu0.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&sdhc0 {
+	status = "okay";
+	mmc: mmc {
+		status = "okay";
+	};
```

Comment:
Move to `max32666fthr_max32666_cpu0.dts` and then you can remove the overlay file